packages/SwingSet/test/workers/vat-target.js

```diff
@@ -15,6 +15,7 @@ function ignore(p) {
 
 export function buildRootObject(vatPowers, vatParameters) {
   console.log(`vat does buildRootObject`); // make sure console works
+  console.log(BigInt(0)); // make sure BigInt is tolerated: #2936
```

Comment:
I just thought of a second test to add: `console.log` should not harden the arguments (which I've accidentally violated at least once). Please add something like:

```js
const mutable = [];
console.log('list:', mutable);
mutable.push(1);
console.log('list+1:', mutable);
```

to make sure I don't make that mistake again.